src/app/frontend/cluster/cluster.html

```diff
@@ -14,6 +14,19 @@
 limitations under the License.
 -->
 
+<div layout="row">
+  <kd-graph-card graph-title="[[CPU usage|Title for graph card displaying CPU metric of one all resources.]]"
+                 metrics="::$ctrl.cluster.nodeList.cumulativeMetrics"
```

Comment:
I think that this PR adds not only sorting. Can you outline all changes in the PR message?